x-pack/plugins/apm/server/ui_settings.ts

```diff
@@ -8,25 +8,24 @@ import { schema } from '@kbn/config-schema';
 import { i18n } from '@kbn/i18n';
 import { UiSettingsParams } from '../../../../src/core/types';
 import {
-  enableSignificantTerms,
+  enableCorrelations,
   enableServiceOverview,
 } from '../common/ui_settings_keys';
 
 /**
  * uiSettings definitions for APM.
  */
 export const uiSettings: Record<string, UiSettingsParams<boolean>> = {
-  [enableSignificantTerms]: {
+  [enableCorrelations]: {
     category: ['observability'],
     name: i18n.translate('xpack.apm.enableCorrelationsExperimentName', {
-      defaultMessage: 'APM Significant terms (Platinum required)',
+      defaultMessage: 'APM Correlations (Platinum required)',
     }),
     value: false,
     description: i18n.translate(
       'xpack.apm.enableCorrelationsExperimentDescription',
       {
-        defaultMessage:
-          'Enable the experimental Significant terms feature in APM',
+        defaultMessage: 'Enable the experimental Correlations feature in APM',
```

Comment:
I don't think this should be capitalized.

Reply:
Not a native speaker but shouldn't it be capitalised if it's a feature name? It's not just "_a_ correlation". It's _the_ Correlation feature. 

Either way, not a big deal :)